web/book/src/queries/variables.md

```diff
@@ -1,11 +1,37 @@
-# Variables
+# Variables and `let`
 
-We can define a relation — similar to a CTE in SQL — with two approaches — a
-prefix `let` or a postfix `into.
+`let` assigns a scalar value, a function, an array, or a pipeline to a named
+variable.
```

Comment:
`let` can be also assigned a tuple, function call or a range.

I think we can just say "any expression, including functions".